ForceFeedback/xUnitTest/ConfigPathTest.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace xUnitTest
+{
+    public class ConfigPathTest
+    {
+		[Theory]
+		//[InlineData("I:\\Work\\49_c# algorithm\\Source\\ThreeProb\\ThreeProb\\HashIterator.cs")]
+		[InlineData("C:\\projects\\forcefeedbackprogramming\\ForceFeedback\\ForceFeedback.Rules\\Configuration\\Global.cs")]
```

Comment:
Do not use absolute paths in tests! This test, for example, cannot possibly work on my machine; I'm using a Mac which does not know about drives (e.g. `C:`).

Reply:
I will use relative path of this project then.